### PR TITLE
 Assignment 2.1 and 2.2 DONE

### PR DESCRIPTION
**Pull request by sutariyaparas**

### CODE REVIEW

File: Week-03/Assignment3.3/scrip.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// closer Example
+// JavaScript variables can belong to the local or global scope.
+// Global variables can be made local (private) with closures.
+
+
+function createIncrement() {
```
Comment (sharmajitesh07) [inline Week-03/Assignment3.3/scrip.js]: use arrow functions

File: Week-03/Assignment3.2/call_fn.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+const customerName = customer.fullName.call(customer1);
+console.log(customerName);
+
+///////////
+// The call() method can accept arguments:
+const person = {
+  fullName: function (city, country) {
+    return this.firstName + " " + this.lastName + "," + city + "," + country;
```
Comment (sharmajitesh07) [inline Week-03/Assignment3.2/call_fn.js:29]: you can use string literal here
return `${this.firstName} ${this.lastName}, ${city}, ${country}`;

**Review by sharmajitesh07 — APPROVED** [review]

File: Week-03/Assignment3.3/scrip.js (hunk 1 above)
Comment (sutariyaparas) [reply Week-03/Assignment3.3/scrip.js]: Done applied

File: Week-03/Assignment3.2/call_fn.js (hunk 2 above)
Comment (sutariyaparas) [reply Week-03/Assignment3.2/call_fn.js:29]: Done applied

**Pull request merged**